tests/wamr-test-suites/test_wamr.sh

```diff
@@ -856,12 +812,73 @@ function collect_coverage()
     fi
 }
 
+# decide whether execute test cases in current running mode based on the current configuration or not
+# return 1 if the test case should be executed, otherwise return 0
+function do_execute_in_running_mode()
+{
+    local RUNNING_MODE="$1"
+
+    # wasm64(memory64) is only enabled in interp and aot mode(?)
+    # FIXME: add "aot" when wamrc supports --enable-memory64 switcher
+    if [[ ${ENABLE_MEMORY64} -eq 1 ]]; then
+        if [[ "${RUNNING_MODE}" != "classic-interp" ]]; then
+            return 0
+        fi
+    fi
+
+    # multi-module only enable in interp mode and aot mode
+    if [[ ${ENABLE_MULTI_MODULE} -eq 1 ]]; then
+        if [[ "${RUNNING_MODE}" != "classic-interp" \
+                && "${RUNNING_MODE}" != "fast-interp" ]]; then
```

Comment:
Should also add aot, `&& "${RUNNING_MODE}" != "aot"`

Reply:
*aot* + *multi-modules* will fail the linking test case. Tend to fix it in another PR.